src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo Elmar</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  
+  <div class="flex-wrapper">
+    <h1>Today's To Do</h1>
+    <div class="add-todo-wrapper">
+      <div class="input-wrapper">
+        <input type="text" id="description-task" placeholder="Add to your list">
+      </div>
+      <div class="button-wrapper">
+        <button id="add-task-btn" type="button">Add</button>
+      </div>
+    </div>
```

Comment:
I think that semantic tags could be used here instead of div. This helps in web accessibility. You could use section, aside, or article tags and still retain every benefit you could get by using the div tag.

Reply:
I fixed it with using <section> tag. Thanks!